### PR TITLE
Pan gesture handling for scroll view

### PR DESCRIPTION
**Pull request by npriyadarshi**
### Platforms Impacted
- [ ] iOS
- [ ] macOS

### Description of changes

(a summary of the changes made, often organized by file)

### Verification

(how the change was tested, including both manual and automated tests)

| Before                                       | After                                      |
|----------------------------------------------|--------------------------------------------|
| Screenshot or description before this change | Screenshot or description with this change |

### Pull request checklist

This PR has considered:
- [ ] Light and Dark appearances
- [ ] VoiceOver and Keyboard Accessibility
- [ ] Internationalization and Right to Left layouts
- [ ] Different resolutions (1x, 2x, 3x)
- [ ] Size classes and window sizes (iPhone vs iPad, notched devices, multitasking, different window sizes, etc)

### CODE REVIEW

File: ios/FluentUI/BottomSheet/BottomSheetViewController.swift [hunk 1]
```diff
@@ -242,3 +243,18 @@ public class BottomSheetViewController: UIViewController {
         }
     }
 }
+
+extension BottomSheetViewController: UIGestureRecognizerDelegate {
+    public func gestureRecognizer(_ gestureRecognizer: UIGestureRecognizer, shouldRecognizeSimultaneouslyWith otherGestureRecognizer: UIGestureRecognizer) -> Bool {
+        let shouldRecognizeSimultaneously = false
```
Comment (harrieshin) [inline ios/FluentUI/BottomSheet/BottomSheetViewController.swift]: why even have this variable if it let and you are just going to return in line 258

File: ios/FluentUI/BottomSheet/BottomSheetViewController.swift [hunk 2]
```diff
@@ -242,3 +243,18 @@ public class BottomSheetViewController: UIViewController {
...
+        if let contentView = otherGestureRecognizer.view {
```
Comment (harrieshin) [inline ios/FluentUI/BottomSheet/BottomSheetViewController.swift]: we should be able to combine these if statements right?

File: ios/FluentUI/BottomSheet/BottomSheetViewController.swift [hunk 3]
```diff
@@ -242,3 +243,18 @@ public class BottomSheetViewController: UIViewController {
...
+            if let scrollView = contentView as? UIScrollView {
+                // If scroll view has reached the bottom or top bring the bottom sheet pan in action too.
+                if scrollView.contentOffset.y >= (scrollView.contentSize.height - scrollView.frame.size.height) || (scrollView.contentOffset.y <= 0) {
```
Comment (harrieshin) [inline ios/FluentUI/BottomSheet/BottomSheetViewController.swift]: if so, we should probably check if othergestureregonizer is a pangesture to begin with

File: ios/FluentUI/BottomSheet/BottomSheetViewController.swift (hunk 2 above)
Comment (harrieshin) [reply ios/FluentUI/BottomSheet/BottomSheetViewController.swift]: if let scrollView = otherGestureRecognizer.view as? UIScrollView, scrollView.contentOffset.y >= (scrollView.contentSize.height - scrollView.frame.size.height) || (scrollView.contentOffset.y <= 0) {

File: ios/FluentUI/BottomSheet/BottomSheetViewController.swift (hunk 3 above)
Comment (npriyadarshi) [reply ios/FluentUI/BottomSheet/BottomSheetViewController.swift]: this makes sense

File: ios/FluentUI/BottomSheet/BottomSheetViewController.swift (hunk 2 above)
Comment (npriyadarshi) [reply ios/FluentUI/BottomSheet/BottomSheetViewController.swift]: i will update this

File: ios/FluentUI/BottomSheet/BottomSheetViewController.swift (hunk 1 above)
Comment (npriyadarshi) [reply ios/FluentUI/BottomSheet/BottomSheetViewController.swift]: Will update this.

Comment (npriyadarshi) [reply ios/FluentUI/BottomSheet/BottomSheetViewController.swift]: i didn;t realise it was not using it, before raising the PR...my bad

**Pull request merged**